index.js

```diff
@@ -169,10 +176,61 @@ module.exports = (robot) => {
     authorize(JSON.parse(clientSecret), checkClaSheet);
   };
 
+  var checkMergeConflicts = async function(context) {
+    var mergeConflictLabel = ['PR: don\'t merge - HAS MERGE CONFLICTS'];
+    pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+      context.repo());
+
+    arrayOfOpenPullRequests = pullRequestsPromiseObj.data;

```

Comment:
is it possible to pull the data you want here to avoid having the await in the for loop?

Reply:
Kevin, each of the `await` statements make a call to a GitHub API which is unique for each PR, in the for loop. Removing any of these results in a pending promise due to which the later code is executed first, which we do not want.